src/llm/agent-evals/slack-agent/test-data.ts

```diff
@@ -0,0 +1,270 @@
+  },
+  {
+    description: 'Get all users in the Slack workspace.',
+    chat_history: [],
+    invocation: {
+      initiator_name: 'John_Snow',
+      message: '@Quix Who are all the users in this workspace?'
+    },
+    reference_tool_calls: [
+      {
+        name: 'slack_get_users',
+        arguments: {
+          limit: 100
+        }
+      }
+    ],
+    expected_response: 'Workspace users: John, Obreyn, Robb, Jamie, SystemBot, Alice.',
+    tool_mock_response_overrides: {}
+  },
+  {
+    description:
+      'Add a :thumbsup: reaction to the latest message in #general, requiring channel history fetch first.',
+    chat_history: [
+      { author: 'John_Snow', message: 'Great job on the release!' },
+      { author: 'Alice_Hightower', message: 'Thanks everyone!' }
+    ],
+    invocation: {
+      initiator_name: 'John_Snow',
+      message: '@Quix Add a thumbs up reaction to the latest message in #general'
+    },
+    reference_tool_calls: [
+      {
+        name: 'slack_list_channels',
+        arguments: {
+          limit: 100,
+          cursor: ''
+        }
+      },
+      {
+        name: 'slack_get_channel_history',
+        arguments: {
+          channel_id: 'C134DSD',
+          limit: 10
+        }
+      },
+      {
+        name: 'slack_add_reaction',
+        arguments: {
+          channel_id: 'C134DSD',
+          timestamp: '1716282000.000001',
+          reaction: 'thumbsup'
+        }
+      }
+    ],
+    expected_response: 'Added :thumbsup: reaction to the latest message in #general.',
+    tool_mock_response_overrides: {}
+  },
+  {
+    description: 'Join #project-x and post a message after joining.',
+    chat_history: [
+      {
+        author: 'Jamie_Lannister',
+        message:
+          'We should have a project kickoff meeting at 3pm to discuss the new project in #project-x.'
+      }
+    ],
+    invocation: {
+      initiator_name: 'Jamie_Lannister',
+      message: '@Quix Join #project-x and post about the project kickoff meeting'
+    },
+    reference_tool_calls: [
+      {
+        name: 'slack_list_channels',
+        arguments: {
+          limit: 100,
+          cursor: ''
+        }
+      },
+      {
+        name: 'slack_join_channel',
+        arguments: {
+          channel_id: 'C874HKJ'
+        }
+      },
+      {
+        name: 'slack_post_message',
+        arguments: {
+          channel_id: 'C874HKJ',
+          text: 'We have a project kickoff meeting at 3pm. Please join if you can.'
+        }
+      }
+    ],
+    expected_response:
+      "Joined #project-x and posted: 'We have a project kickoff meeting at 3pm. Please join if you can.'",
+    tool_mock_response_overrides: {}
+  },
+  {
+    description: 'Reply to a thread in #project-x channel in the discussion about login issues.',
+    chat_history: [
+      { author: 'John_Snow', message: 'Can someone help with the login issue?' },
+      { author: 'Slack_Bot', message: 'Check the error logs in #project-x.' }
```

Comment:
Again same reason as mentioned elsewhere using any random name works
```suggestion
      { author: 'Jamie Lannister', message: 'Check the error logs in #project-x.' }
```